rdr_service/model/study_nph_sms.py

```diff
@@ -64,7 +64,7 @@ class SmsSample(NphBase):
     sample_identifier = Column(String(32))
     diet = Column(String(32))
     sex_at_birth = Column(String(32))
-    bmi = Column(String(4))
+    bmi = Column(String(5))
```

Comment:
TBH knowing the BB, I would definitely extend this to at least `32`, just to be on the safe side, if they decide to start sending floats for this. I see you have validation for that but sometimes its easier to defensively program against what the partners are going to send.